app/services/api/v3/actors/sustainability_table.rb

```diff
@@ -135,6 +136,26 @@ def totals_row(group_totals_hash)
               end
           }
         end
+
+        def get_tooltip(ro_chart_attribute)
```

Comment:
I guess we'll need the same logic for some of the place profile charts, so I'd consider moving this function to its own service / module under the `Api::V3::Profiles` namespace for example (might also make it easier to write specs)

Reply:
good idea about extracting this. I added a separate module in the next commit, will add specs to it in another one